region/norfair/east/Bubble Mountain.json

```diff
@@ -2992,11 +3217,32 @@
         "h_iceClip"
       ],
       "flashSuitChecked": true,
+      "blueSuitChecked": true,
       "note": [
         "Freeze the wall crawler at a precise location in order to jump through the Power Bomb Blocks.",
         "The pixel window is larger and higher with Morph and an X-Ray Stand Up."
       ]
     },
+    {
+      "link": [5, 9],
+      "name": "G-Mode Morph with Blue Suit",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        },
+        "comesThroughToilet": "any"
+      },
+      "requires": [
+        {"haveBlueSuit": {}},
+        {"or": [
+          "h_artificialMorphIBJ",
+          "h_artificialMorphSpringBall"
+        ]}
+      ],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true
+    },
```

Comment:
```suggestion
```
this wont work in direct, as you'll fall through. It won't work in indirect, because there's no downward setup here and no way to doorlock bypass while morphed and no up door is missing a door cap ... unless we were wanting to support asymmetric door connections or the possibility of coming from a non-vanilla room (with no doorcap)